assets/src/Routes.tsx

```diff
@@ -1,9 +1,9 @@
 import React, { FC } from "react";
 import RoomPage from "./pages/room/RoomPage";
-import { HomePage } from "./pages/home/HomePage";
```

Comment:
You can remove `HomePage` because it is unused.

Reply:
it's already removed